posts/2023-03-31-embrace-kotlin.adoc

```diff
@@ -0,0 +1,273 @@
+= Embrace Kotlin: Where to Get Started for Scala Developers
+AnastasiiaL
+v1.0, 2023-03-31
+:title: Embrace Kotlin: Tips & Tricks for Scala Developers to Get Started
+:imagesdir: ../media/2023-03-31-embrace-kotlin
+:lang: en
+:tags: [scala, kotlin]
+
+Recently I have been intrigued by the buzz surrounding Kotlin, another powerful language on the JVM.
+The growing popularity and claims of a seamless transition from Scala to Kotlin have piqued my curiosity,
+prompting me to dive in and explore the language myself.
+In this blog post, I'll share my journey of learning Kotlin from a Scala developer's perspective and provide
+insights into the similarities and differences between the two languages that make the switch feel so smooth.
+
+== Language similarities
+
+Both Kotlin and Scala are statically-typed languages that run on the Java Virtual Machine. They share many features, such as:
+- Support for both functional programming and OO
+- Interoperability with Java
+- Type inference
+- Null safety
+- Extension functions
+- Pattern matching (although more limited in Kotlin)
+These similarities create a strong foundation for Scala developers to quickly grasp Kotlin's syntax and paradigms.
+For me personally it was quite easy to create my first kotlin app without looking up "how-to"s, because structurally there would be nothing new.
```

Comment:
```suggestion
For me personally it was quite easy to create my first Kotlin app without looking up "how-to"s, because structurally there would be nothing new.
```
Missing initial capital letter